db/migrate/20200116162348_create_orders.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateOrders < ActiveRecord::Migration[6.0]
```

Comment:
?

Reply:
Its the very last step. Ma fault. Deleted.